packages/drivers/local-driver/package.json

```diff
@@ -33,7 +33,9 @@
     "test:mocha:verbose": "cross-env FLUID_TEST_VERBOSE=1 npm run test:mocha",
     "tsc": "tsc",
     "tsfmt": "tsfmt --verify",
-    "tsfmt:fix": "tsfmt --replace"
+    "tsfmt:fix": "tsfmt --replace",
+    "typetests:gen": "fluid-type-validator -m packages -v",
```

Comment:
Are these the right options? Just checking since they're different than almost all the others.

Reply:
No, no they're not. Thanks for catching this.